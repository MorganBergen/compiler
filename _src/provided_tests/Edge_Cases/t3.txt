TASKS
    2 5
POLY
    H = x x x;
EXECUTE
    INPUT V;
    V = H(V);
    OUTPUT V;
INPUTS
    3
