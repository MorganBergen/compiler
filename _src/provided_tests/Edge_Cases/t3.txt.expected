27
H: 3
